bld/CLMBuildNamelist.pm

```diff
@@ -4751,8 +4765,6 @@ sub setup_logic_fates {
               my $fname = remove_leading_and_trailing_quotes( $nl->get_value($var) );
               if ( ! defined($nl->get_value($var))  ) {
                  $log->fatal_error("$var is required when use_fates_inventory_init is set" );
-              } elsif ( ! -f "$fname" ) {
```

Comment:
I removed the check for file existence here, because it was too early and caused problems. The check for file existence is a general thing that should be done for all filenames anyway, so doesn't need to have special code to handle it for each one here.

However, this now is still failing to check for file existance at the right time, which causes a build-namelist test to fail. I'm looking into this now...

Reply:
Ahh, the file existence checking is in add_default and these filenames are in the testmods rather than going into the namelist_defaults. So I'll make an issue to move them to namelist_defaults, which will then take care of checking for existance.

The other check for file existence happens in a case with the check_input_data step. So it will fail at that point if the file doesn't exist.